corehq/apps/users/tasks.py

```diff
@@ -65,16 +63,24 @@ def tag_forms_as_deleted_rebuild_associated_cases(self, formlist, deletion_id,
         cases_to_rebuild.update(get_case_ids_from_form(form))
     XFormInstance.get_db().bulk_save(formlist)
 
+    for case in cases_to_rebuild - deleted_cases:
+        _rebuild_case_with_retries.delay(case)
```

Comment:
Do we want a single task per case or should we use something like celery map / chunks: http://celery.readthedocs.org/en/latest/userguide/canvas.html#chunks